### ISSUE TITLE
issue titles have insufficient contrast

### ISSUE THREAD

**Issue by thatbudakguy** [Princeton-CDH/startwords#120]
**Describe the bug**
The issue titles on both the issue list page and single issue page have a contrast ratio with the background of **1.82**, which fails [WCAG AA standards](https://www.w3.org/TR/WCAG21/#contrast-minimum) for large text (minimum **3.0**).

**To reproduce**
See [this contrast ratio calculator](https://contrast-ratio.com/#rgb%28128%2C%20128%2C%20128%2C%200.5%29-on-white) that takes into account `rgba()` colors with transparency. Other contrast calculators may incorrectly ignore the fact that the color of the titles is transparent - `rgba(128, 128, 128, 0.5)`.

**Expected behavior**
All text on the site should pass WCAG AA requirements.

**Screenshots**
taken using firefox's accessibility panel in the developer tools.
<img width="319" alt="Screen Shot 2020-09-01 at 5 11 30 PM" src="https://user-images.githubusercontent.com/4924494/91906464-37854b80-ec76-11ea-9a8a-761ac9c40325.png">

**Additional context**
The non-transparent version of the color - `rgb(128, 128, 128)` - _does_ pass WCAG requirements, with a contrast ratio of about **3.9**.
